File: exam_report/examshell/level2/do_op/do_op.c
#include "do_op.h"

int main(int argc, char **argv)
{
	int result;

	if (argc == 4)
	{
		if (*argv[2] == '+' || *argv[2] == '-' || *argv[2] == '/' || \
				*argv[2] == '*' || *argv[2] == '%')
		{
			if (*argv[2] == '+')
				result = atoi(argv[1]) + atoi(argv[3]);
			else if (*argv[2] == '-')
				result = atoi(argv[1]) - atoi(argv[3]);
			else if (*argv[2] == '/')
				result = atoi(argv[1]) / atoi(argv[3]);
			else if (*argv[2] == '*')
				result = atoi(argv[1]) * atoi(argv[3]);
			else if (*argv[2] == '%')
				result = atoi(argv[1]) % atoi(argv[3]);
			printf("%d", result);
		}
	}
	printf("\n");
	return (0);
}

File: Piscine_PHP/d01/ex10/do_op.php
#!/usr/bin/php
<?PHP

if ($argc != 4)
	echo "Incorrect Parameters" . PHP_EOL;
else {
	$argv[1] = preg_replace('/\s+/', '', $argv[1]);
	$argv[2] = preg_replace('/\s+/', '', $argv[2]);
	$argv[3] = preg_replace('/\s+/', '', $argv[3]);
	if ($argv[2] == "+")
		echo $argv[1] + $argv[3] . PHP_EOL;
	else if ($argv[2] == "-")
		echo $argv[1] - $argv[3] . PHP_EOL;
	else if ($argv[2] == "*")
		echo $argv[1] * $argv[3] . PHP_EOL;
	else if ($argv[2] == "/")
		echo $argv[1] / $argv[3] . PHP_EOL;
	else if ($argv[2] == "%")
		echo $argv[1] % $argv[3] . PHP_EOL;
}

?>